### ISSUE TITLE
TracerWarning: Converting a tensor to a Numpy array might cause the trace to be incorrect

### ISSUE THREAD

**Issue by Gramet** [teffland/pytorch-monitor#17]
`/usr/local/lib/python3.5/dist-packages/pytorch_monitor/monitor.py:72: TracerWarning: Converting a tensor to a NumPy array might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  param.detach().cpu().numpy(),`

`/usr/local/lib/python3.5/dist-packages/pytorch_monitor/monitor.py:79: TracerWarning: Converting a tensor to a NumPy array might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  update.detach().cpu().numpy(),`

`/usr/local/lib/python3.5/dist-packages/pytorch_monitor/monitor.py:86: TracerWarning: Converting a tensor to a NumPy array might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  update_ratio.detach().cpu().numpy(),`

I'm getting those three every time I start monitoring my model... Does this raises any issues as to how everything is being monitored? If not It would be nice to simply have them remove automatically.

**Comment by teffland** [comment]
This appears to be something to do with the pytorch jit. E.g., [link](https://discuss.pytorch.org/t/torch-jit-trace-tracerwarning-converting-a-tensor-to-a-python-list-might-cause-the-trace-to-be-incorrect/26632).

Are you using this with your model?

As far as I can tell this is outside the scope of this repo. It shouldn't have any impact on the logging itself though I think, so maybe you just want to disable those warnings in your own code.

**Comment by Gramet** [comment]
Yeah I figured it is related to Pytorch jit, I was just wondering if it was something to be concerned about regarding the logging or not. 

The thing is the warning is coming from pytorch-monitor since it is caused by `param.detach().cpu().numpy()`. If it is expected behavior to break the Trace here and the user should not worry about it, I think the warning should be muted directly in there so it's invisible to the user and nobody will notice it. 

Anyway, I muted it in my code for now. Thank you for your quick reply :)
